### ISSUE TITLE
Handle 2.3 prettier changes

### ISSUE THREAD

**Issue by adamzapasnik** [adamzapasnik/prettier-plugin-erb#49]

**Comment by Juice10** [comment]
Issues thanks to incompatibility of prettier 2.3 and above: https://github.com/adamzapasnik/prettier-plugin-erb/issues/97 https://github.com/adamzapasnik/prettier-plugin-erb/issues/44 https://github.com/adamzapasnik/prettier-plugin-erb/issues/97

**Comment by amkisko** [comment]
Here are some upgrades to 2.8.0, but still failing in tests:
https://github.com/amkisko/prettier-html-templates - all the issues are visible in tests now
https://github.com/amkisko/prettier-plugin-erb - this one is more complex due to changes in prettier and at the moment in tests it always returns 0 bytes output

UPD 27.01.2024: these repositories are removed.

**Comment by tmaier** [comment]
@amkisko great to see this. Does it already make sense for you to share a Work-in-progress (WIP) Pull Request? Maybe in collaboration with @Juice10, who also started a similar Pull Request

**Comment by amkisko** [comment]
@tmaier @Juice10 you are welcome:
- https://github.com/adamzapasnik/prettier-html-templates/pull/56
- https://github.com/adamzapasnik/prettier-plugin-erb/pull/135

It seems that there are some complex problems to solve. :)

**Comment by drewlustro** [comment]
> Here are some upgrades to 2.8.0, but still failing in tests: [amkisko/prettier-html-templates](https://github.com/amkisko/prettier-html-templates) - all the issues are visible in tests now [amkisko/prettier-plugin-erb](https://github.com/amkisko/prettier-plugin-erb) - this one is more complex due to changes in prettier and at the moment in tests it always returns 0 bytes output

Hi @amkisko – why did you delete the [amkisko/prettier-html-templates](https://github.com/amkisko/prettier-html-templates) fork?

Did you give up and `prettier-plugin-html-erb` is the replacement?

**Comment by amkisko** [comment]
Yes, I did, although PRs are still available, no more changes happened after publishing.

There are few better options available:
- erb-lint (which also adds support for rubocop which is very useful)
- erb-formatter
- rewriting https://github.com/ecmel/prettier-plugin-ejs to support erb, which I did not continue too
